index.js

```diff
@@ -145,16 +145,34 @@ function $asInteger (i) {
 }
 
 function $asNumber (i) {
+  if (i === 0) return '0'
+  if (!i || i === true) {
```

Comment:
this check would be very slow to perform on turbofan.

Reply:
Remove this `if`